Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -123,6 +123,28 @@ public static class TestObjects {
         public static Teacher drFord = new Teacher("Rachel", "Ford", "Chemistry", DateTime.Parse("04/29/1985"));
         private static Teacher[] faculty = { drKyler, drFord };
         public static School fieldAcademy = new School("1234 Fictional Ave", 150, new List<Teacher>(faculty));
+
+        public static PoliticalParty democrats = new PoliticalParty { Name = "Democratic Party" };
```

Comment:
Please remove any political references of any kind from the code.

Reply:
Addressed.